src/reducers/queriesReducer.js

```diff
@@ -45,6 +45,11 @@ export default function queriesReducer(state = {}, action) {
       case actionTypes.DOCUMENT_MODIFIED:
         set(draft, [key, 'data', action.meta.doc], action.payload.data);
         return draft;
+      case actionTypes.MULTIPLE_DOCUMENTS_ADDED:
+        action.payload.forEach(changeAction => {
+          set(draft, [key, 'data', changeAction.meta.doc], changeAction.payload.data);
+        })
+        return draft;
```

Comment:
Not postive, but I think we should be returning a copy here

Reply:
@prescottprue hey, I was just following the way it's done in other cases. For ex. a few lines above, you modify draft directly